encryption_works.md

```diff
@@ -98,25 +98,58 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
+### What Tor Doesn't Protect You From
 
 Documentation for Tails, the live GNU/Linux distribution that forces all of the user's network traffic to go through the Tor network, has [this to say](https://tails.boum.org/doc/about/warning/index.en.html#index7h1) about global adversaries:
 
 > A global passive adversary would be a person or an entity able to monitor at the same time the traffic between all the computers in a network. By studying, for example, the timing and volume patterns of the different communications across the network, it would be statistically possible to identify Tor circuits and thus matching Tor users and destination servers.
 
-We still don't know whether or not NSA or GCHQ counts as a global adversary, but we do know that they monitor a large portion of the Internet. It's too early to know for sure how often these intelligence agencies can defeat the anonymity of the Tor network.
+We don't know if the NSA and GCHQ are big enough to fit this definition of "global adversities," but we do know they monitor a large portion of the Internet. It's too early to say whether this large-scale surveillance is enough to defeat Tor's network anonymity in the general sense.
 
-Even if they can, using Tor still gives us many advantages. It makes their job much harder, and we leave much less identifying data on the servers we connect to through the Tor network. It makes it much harder to be the victim of a MITM attack at our local network or ISP level. And even if some Tor circuits can be defeated by a global adversary, if enough people are getting their traffic routed through the same Tor nodes at the same time, it might be difficult for the adversary to tell which traffic belongs to which circuits.
+Leaked NSA slides [have](http://www.theguardian.com/world/interactive/2013/oct/04/tor-stinks-nsa-presentation-document) titles such as "Tor Stinks" and have [called](http://www.theguardian.com/world/interactive/2013/oct/04/tor-high-secure-internet-anonymity) Tor "Still the King of high secure, low latency Internet Anonymity" with "no contenders."
 
-The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en).
+That said, leaked NSA slides indicate the surveillance agency can de-anonymize "a very small fraction" of Tor users. 
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/torbrowser.png)
+An attack like this becomes much harder as the number of nodes and the volume of traffic increases -- not impossible, but more time-consuming and expensive. If you're a student or staff-member at a college or university, the Electronic Frontier Foundation encourages you to [set up](https://www.eff.org/torchallenge/tor-on-campus.html) a Tor node on campus.
```

Comment:
It's good to mention that specific advice, but you don't have to be at a university to run a Tor relay! Everyone should do it :)